### ISSUE TITLE
Analytics are tracked without player agreement

### ISSUE THREAD

**Issue by PG-Fuury** [Blitex/moreplayermodels#1]

**Comment by adamc295** [comment]
Pinging to get attention from the creator...
@Blitex

**Comment by Storm-Cloud** [comment]
@adamc295 @PG-Fuury 

Have you noticed that the entire code base was copied in from a single commit? And this isn't the mod author's known github. It's obvious the Blitex didn't code any of this.

**Comment by Blitex** [comment]
> @adamc295 @PG-Fuury
> 
> Have you noticed that the entire code base was copied in from a single commit? And this isn't the mod author's known github. It's obvious the Blitex didn't code any of this.



> Pinging to get attention from the creator...
> @Blitex
> 
> Edit:
> It's been 3 days and the creator hasn't responded. Not suspicious at all... \s



> _No description provided._

Yes this is mod isn't mine, the author gave me the source code a long time ago when I was trying to add my own 3D models as playable models. I gave up on it a while ago due to me not being able to achieve my goal, and forgot to take it down. I am not attempting to pass this mod as my own, it is on GitHub because of a coding tutorial I was watching. Sorry for any misunderstandings.

**Comment by Blitex** [comment]
If you see my latest commit you will see that it is name "mod for obj." That was me trying to add my own OBJ models.
